### ISSUE TITLE
Request Project Annotation [ PXD006003 ]

### ISSUE THREAD

**Issue by jgriss** [bigbio/proteomics-metadata-standard#306]
- Project Accession: [ PXD006003 ]
- URL Project: [ https://www.ebi.ac.uk/pride/archive/projects/PXD006003 ] 
- PubMed Accession: [ https://pubmed.ncbi.nlm.nih.gov/31495571 ]

**Comment by javizca** [comment]
I think it would make sense to see whether Expression Atlas has some example already, and follow the same way. I guess it needs to be clear that it is a xenograph somewhere. Species should be both human and mouse I guess?

And in my view it makes sense to use the clinical metadata from the patient.

**Comment by jgriss** [comment]
@StSchulze @levitsky @ypriverol Continuing our discussion about #223 how should we annotate the source of the PDX.

In this project, the authors analysed both the primary tumour as well as the PDX. Keeping track of this relationship might be interesting. Is this something we want to support?

**Comment by mlocardpaulet** [comment]
The field `characteristics[individual]` should allow you to match the PDX to the primary tumour, don't you think?

**Comment by jgriss** [comment]
@mlocardpaulet So you suggest to put the `sample name` in `characteristics[individual]` for PDX samples and "none" in the primary patient samples?

**Comment by mlocardpaulet** [comment]
No, sorry I wasn't clear. I think `characteristics[individual]` should be the same for the PDX and primary tumour. That is what allows their matching. 
For "PDX" Vs "primary tumour" you need an other field. That would be the case for any of these types of experiment. I guess you could also have "primary tumour" and "lung metastasis" in an other scenario... Unfortunately, I am not really familiar with the annotation fields that you have at disposal for the sdrf headers... So I don't know which one you can use to have "primary tumour", "PDX", ... Maybe it could be `organism part`? Or maybe the EFO term `sampling site`?

**Comment by anjaf** [comment]
I agree that the source should be annotated as the donor organism (i.e. the cancer patient). The xenograft is usually only the means to growth the specimen. Unless the researchers are particularly interested in the host's immune response or similar (then the mouse would be source organism). 
We would annotate the fact that it's a PDX, under the term `Characteristics[xenograft]` or as `Characteristics[growth condition]`. 
Here are a few examples from ArrayExpress. They are not overly curated (as they are not in Expression Atlas) but this may give you an idea. 
https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5006/samples/
https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-7568/samples/

Regarding the primary vs PDX samples, yes, including `Characteristics[individual]` with the same (donor) ID is the best way to capture the relationship between the samples.

**Comment by jgriss** [comment]
@anjaf Thanks a lot for this information! I will follow this approach.

@ypriverol I suggest that we explicitly mention this in the specification. Should I add this and create a PR?

**Comment by ypriverol** [comment]
Yes @jgriss. In that way, we have more people contributing to the Specification. 👍
